facades/forecast.js

```diff
@@ -0,0 +1,31 @@
+var User = require('../models').User;
+var GeocodeService = require('../services/geocode');
+var ForecastService = require('../services/forecast');
+
+module.exports = class ForecastFacade {
+  constructor(status, body) {
```

Comment:
This is the first file I'm looking at, and I'm going to keep digging in, but off the top I'm not super clear on what "status" would be for a ForecastFacade. It's possible that you've already found the most descriptive term there, but if not, is there anything more descriptive you could use?

Reply:
Ah, the request status, I get it now.